components/button-stateful/__tests__/button-stateful.browser-test.jsx

```diff
@@ -11,10 +11,13 @@ chai.should();
 
 describe('Button Stateful: ', () => {
 	// Base defaults
-	const requiredProps = {
+	const requiredPropsNoVariant = {
```

Comment:
To avoid creating an addition variable here, you could go with the original 
```
const requiredPropsNoVariant = {
	assistiveText: { icon: 'like' },
	iconName: 'like',
	iconSize: 'large',
	variant: 'icon'
};
```
and then switch the `assign` call arguments below so that the custom props overwrite values in `requiredProps`.

Reply:
I think it is more readable and maintainable to keep the 2 set of required values, and keep them fixed.  In fact, the switch in the assign() was a deliberate test fix because the overwritten value ended up interfering each other in the test runs when I was extending the tests.